src/Controller/HomeController.php

```diff
@@ -8,10 +8,17 @@
 
 class HomeController 
 {
+    protected $app;
+
+    public function __construct(App $app)
+    {
+        $this->app = $app;
+    }
+
     public function home()
     {
         $content = Trending::fetch($this->getArgs());
-        $offset = $_GET['offset'] ?? App::get('config')['offset'];
+        $offset = $_GET['offset'] ?? $this->app->registry['config']['offset'];
```

Comment:
>  not sure if that's good at all .. seems unreadable to me :D

Only because of the current implementation 😛 Let's wait for the next iteration, maybe you'll come up with a better idea.